pulsar-io/debezium/core/pom.xml

```diff
@@ -62,12 +62,6 @@
       <version>${kafka-client.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>pulsar-client-original</artifactId>
```

Comment:
This is very interesting.
it looks like that we do not need to import the pulsar-client-original anymore